docs-chef-io/content/supermarket/config_rb_supermarket.md

```diff
@@ -53,6 +53,8 @@ This configuration file has the following general settings:
 `default['supermarket']['fqdn']`
 
 : The fully qualified domain name for the Supermarket server. Defaults to using the current FQDN for the machine.
+: For allowing multiple hostname in `default['supermarket']['fqdn']`, specify `node['hostname']` separated by coma.
+Eg. node['https://hostname1.example1.com', https://hostname2.example2.com]
```

Comment:
We should not change anything for the `hostname`. It was only the `allowed_hosts` attribute that needed to be changed. This is a very specific use case which is not so common. So better not to change anything for the `hostname` attribute. We don't know if FQDN can support multiple hostname or not.